docs/_includes/chart.html

```diff
@@ -0,0 +1,602 @@
+<div id="chart">
+    {% include section-header.html name="chart" version=page.versions.chart %}
+    <p>The chart module facilitates the layout of a chart, chart-legend, and any other applicable text (e.g. metric or subtitle text).</p>
+    <h3>Stacked chart with 3 segments</h3>
+    <div class="demo">
+        <div class="demo__inner">
+            <div aria-label="Donut chart title" class="chart chart--stacked">
+                <div tabindex="0" class="donut-graph chart__container chart__container-stacked">
```

Comment:
So this tabindex=0 on the root means a users has to make an additional TAB key press to get into the widget. I think we can remove it so that tabbing lands us directly onto the first slice (similar kb behaviour to a radio button group).

Reply:
👍